testutil/server/configurator/configurator.go

```diff
@@ -0,0 +1,117 @@
+// Package configurator a Fixture defines an implementation of server.Fixture which uses an in-memory test data store
+// with no backing ABCI app. It implements the module.Configurator interface and is designed
+// to be used in server integration tests for each module independent of a larger app.
+//
+// Ex:
+//
+package configurator
```

Comment:
Can't we just use `types/module/server/testutil.go` instead of this whole package? This looks like code that already got removed

Reply:
Well the group module is kind of in an hybrid state at the moment since it doesn't really use the new server module manager and modules/services registration introduced by https://github.com/regen-network/regen-ledger/pull/157 because this doesn't handle genesis yet and we need to provide some params `Subspace` when registering the group services (in order to retrieve the max comment size) which is not handled either. That being said, we might just wanna get rid of this `MaxCommentLength` params (cf. https://github.com/regen-network/regen-ledger/pull/154#discussion_r562650879) so maybe we could move the group module to use #157 if we're ok with implementing genesis im/export later. Wdyt @aaronc?

Reply:
We could just have `MaxMetadataLength` be fixed to 256 or 1024 bytes or something for now... What do you think? I don't really have a strong preference, but it seems like migrating to ADR 033 sooner rather than later would reduce code complexity a bit here no? And we'll need to address the `x/params` upgrade at some point soon anyway.

Reply:
Genesis isn't handled in ADR 033 either though right?

Reply:
I believe that just having some fixed `MaxMetadataLength` for now would be ok and it'd allow us to fully migrate the group module to ADR 033 and use `types/module/server/testutil.go` for tests then. We can deal with genesis later (https://github.com/regen-network/regen-ledger/issues/214).